mlflow/entities/run_data.py

```diff
@@ -5,28 +5,33 @@
 
 
 class RunData(_MLflowObject):
+    """
+    Class exposing run data (metrics and parameters).
+    """
     def __init__(self, metrics=None, params=None):
         self._metrics = []
         self._params = []
         if metrics is not None:
             for m in metrics:
-                self.add_metric(m)
+                self._add_metric(m)
         if params is not None:
             for p in params:
-                self.add_param(p)
+                self._add_param(p)
 
     @property
     def metrics(self):
+        """List of :py:class:`mlflow.entities.metric.Metric` for the current run."""
         return self._metrics
 
     @property
     def params(self):
+        """List of :py:class:`mlflow.entities.param.Param` for the current run."""
         return self._params
 
-    def add_metric(self, metric):
+    def _add_metric(self, metric):
```

Comment:
Made private since these are used in the constructor / aren't intended to be called by users